index.js

```diff
@@ -146,7 +147,7 @@ class TuyaDevice extends EventEmitter {
     return new Promise((resolve, reject) => {
       // Send request
       this._send(buffer).then(async data => {
-        if (data === 'json obj data unvalid' && options.schema !== true) {
+        if (data === 'json obj data unvalid' && options.schema === true) {
```

Comment:
Why that change?

Reply:
honestly, the only reason is when i issue a normal get request to a non-zigbee device through https://github.com/TuyaAPI/cli/blob/master/lib/control.js#L49 (where schema is set to true) it doesn't send a null set request to get the status and it just resolves to "json obj data unvalid". i didn't really understand what the schema option should be doing reading through the code...

Reply:
The `schema` option is a result of poor API design on my part 😛. It controls the data that's passed back through TuyAPI, not the contents of the payload itself.

I'm not entirely sure why we're checking the schema option here; I think that condition should just be dropped.

Reply:
Honestly I also struggled on last PR and "usage" on that one and implemented it by myself outside... so it is a change in behavour if we kill the flag (and I'm up for it!!) but should be clearly stated (but I assume not breaking)

Reply:
Yeah, I don't think removing the check for `schema` here should be breaking. I'm thinking `schema` is still used to control the shape of return data for backwards compatibility.

Reply:
I agree, schema as a flag to send all attributes is good for the moment (we can think about changes in the future, like it doesn't fully make sense to me that the default behaviour is to return only all the dps instead of the whole response -> if i send a request, i want the whole response, not just a part). But I agree that removing the check is necessary. I removed it in the latest commit